model/user.go

```diff
@@ -633,6 +633,10 @@ func (u *User) IsGuest() bool {
 	return IsInRole(u.Roles, SYSTEM_GUEST_ROLE_ID)
 }
 
+func (u *User) IsAdmin() bool {
```

Comment:
```suggestion
func (u *User) IsSystemAdmin() bool {
```
Just wondering if you think we should differentiate it from channel admin and team admin by adding the word System to the method name?

Reply:
Good catch! I've changed it.